addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1581,6 +1582,442 @@ var FieldBinaryImage = AbstractFieldBinary.extend({
     },
 });
 
+var SignatureDialog = Dialog.extend({
```

Comment:
this is not the correct place to define this dialog, in fact the entire definition of the binarySignature widget should be in its own file in the fields folder